lib/capistrano/gitflow/helpers/helper.rb

```diff
@@ -216,5 +216,25 @@ def gitflow_tag_production
       set :branch, new_production_tag
     end
 
+    def gitflow_cleanup_tags
+      return if fetch(:gitflow_keep_tags).nil?
+      tags = `git log --tags  --simplify-by-decoration --pretty="format:%at %D" | grep 'tag:' |sort -n | awk '{$1=""; print $0}' | tr "," "\n" | sed 's/tag:*//' | sed -e 's/^[ \t]*//' |grep -E "[staging|production]{1}-[0-9]{4}-[0-9]{2}-[0-9]{2}\-([0-9]*)"`
```

Comment:
This is pretty tough to read, especially with the series of pipes. Would it be easier to process some of those things in ruby instead?